examples/basics/custom_embeddings_sdk.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
Thought here, just more of a comment, might be good to inform MLSE of this notebook, as they typically manage it.

Reply:
This is a pre-existing notebook, I just updated the examples (as well as some minor text changes for readability) to use the `SDK` as opposed to `advlib`.

https://github.com/Labelbox/labelbox-python/blob/develop/examples/basics/custom_embeddings.ipynb